libs/vkd3d/vkd3d_private.h

```diff
@@ -1415,7 +1416,7 @@ HRESULT vkd3d_create_descriptor_set_layout(struct d3d12_device *device,
         VkDescriptorSetLayoutCreateFlags flags, unsigned int binding_count,
         const VkDescriptorSetLayoutBinding *bindings, VkDescriptorSetLayout *set_layout);
 
-#define VKD3D_MAX_DYNAMIC_STATE_COUNT (7)
+#define VKD3D_MAX_DYNAMIC_STATE_COUNT (8)
```

Comment:
Unrelated to your change but isn't this already wrong and now still wrong?

Before `dynamic_state_list` had 8 elements that could add a VkDynamicState, now it has 9...

https://github.com/HansKristian-Work/vkd3d-proton/blob/621c34a4f3d76849d267619ab6d271db4fa95dce/libs/vkd3d/state.c#L2815